scripts/check-redact-log

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# This script checks if there is plain (not redacted) log in TiKV.
+
+set -e
+
+if grep -r -n -P '(?<!hex_)encode_upper' -C 1 --color=always --include \*.rs --exclude hex.rs --exclude-dir target . ; then
+    echo "To print user data into info logs or error messages, use log_wrappers::Value() instead of hex::encode_upper. The former will respect \`security.redact-info-log\` config and filter out user data from info log if needed. Otherwise, use \`tikv_util::hex_encode_upper\` to get around the lint error. See https://github.com/tikv/tikv/pull/9250 for more information." 
+    exit 1
+else
```

Comment:
Now that `exit` is used, no need to `set -e` and use `else`.

Reply:
I don't see it's modified or replied.

Reply:
I only added `exit 1` and forgot to remove `set -e` and else from this file. Now these are resolved.

Reply:
@BusyJay I would always use `set -e` in a bash script (along with -uo pipefail`). In this case it works fine with or without it. I am not suggesting it needs to be changed back now, just pointing out how to always be defensive with baseh.